src/app/main.tsx

```diff
@@ -2,15 +2,15 @@ import ReactDOM from "react-dom";
 import "./globals.css";
 import WelcomePage from "./pages/WelcomePage/welcomePage";
 import LoginPage from "./pages/LoginPage/loginPage";
-// import Search from "./components/Search/search";
-// import Login from "./components/Login/login";
+import SearchPage from "./pages/SearchPage/searchPage";
 import { BrowserRouter, Route, Routes } from "react-router-dom";
 
 ReactDOM.render(
   <BrowserRouter>
     <Routes>
       <Route path="/" element={<LoginPage />} />
       <Route path="/WelcomePage" element={<WelcomePage />} />
+      <Route path="/SearchPage" element={<SearchPage />} />
```

Comment:
Usually no camel case is used for the paths because hat part of the URL is case sensitive!
I'd also not use the work "page".

```suggestion
      <Route path="/welcome" element={<WelcomePage />} />
      <Route path="/search" element={<SearchPage />} />
```